R/healthcareai.R

```diff
@@ -39,7 +39,7 @@
 #' have passed.}
 #' }
 #' 
-#' @references \url{http://healthcare.ai}
+#' @references \url{http://healthcareai-r.readthedocs.io}
 #' @seealso \code{\link{LinearMixedModelDevelopment}}
```

Comment:
Would you mind putting two links on this help page?

- healthcareai-r.readthedocs.io
- healthcare.ai